### ISSUE TITLE
'NoneType' object has no attribute 'replace'

### ISSUE THREAD

**Issue by programagor** [pbugnion/jupyterlab-sql#102]
Greetings

I'm using Docker for MySQL and Jupyter, and am having difficulties using this extension. I can see the SQL icon and run it, but when I enter my username, password, and database hostname, the only thing I see is `'NoneType' object has no attribute 'replace'`.

Here are the relevant parts of my `docker-compose`:
```
version: "3.0"
services:
  db:
    restart: always
    image: "mariadb"
    volumes:
      - ./db/data:/var/lib/mysql
    networks:
      - db_net
  jupyter:
    restart: always
    build: ./py
    networks:
      - db_net
      - web_internal
```

Here is an excerpt from my Jupyterlab Dockerfile:
```
FROM jupyterhub/jupyterhub

RUN pip install --upgrade pip

RUN conda install jupyterlab
RUN jupyter labextension install @jupyterlab/hub-extension

RUN pip install jupyterlab_sql pip mysqlclient
RUN jupyter serverextension enable jupyterlab_sql --py --sys-prefix
RUN jupyter lab build
```

Versions: Jupyterlab 1.1.4, Python 3.6.9, IPython 7.8.0

**Comment by pbugnion** [comment]
Thanks for reporting this, and apologies for the late reply.

I can reproduce this when I put no database name in the connection URL. For instance, for a database running locally, if I write:

```
mysql://
```

... in the connection URL, I get the same error. However, if I write:

```
mysql:///employees
```

(to connect to an `employees` database, the plugin works as expected.

**Comment by pbugnion** [comment]
Fixed by #108 .
